### PR TITLE
Login

### PR DESCRIPTION
**Pull request by agnesh02**
Implemented feature

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -13,7 +13,12 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.FirebaseTutorial"
         tools:targetApi="31">
-
+        <activity
+            android:name=".HomeActivity"
+            android:exported="false" />
+        <activity
```
Comment (Agnesh-debug) [inline app/src/main/AndroidManifest.xml:19]: Maybe change the login activity as the root ?

File: app/src/main/java/com/example/firebasetutorial/LoginActivity.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }
+        if (pass.equals("")) {
+            etPassword.setError("This field is required");
+            return;
+        }
+
+        auth.signInWithEmailAndPassword(email, pass).addOnSuccessListener(new OnSuccessListener<AuthResult>() {
+            @Override
+            public void onSuccess(AuthResult authResult) {
+                Toast.makeText(getApplicationContext(), "User logged in successfully", Toast.LENGTH_SHORT).show();
```
Comment (Agnesh-debug) [inline app/src/main/java/com/example/firebasetutorial/LoginActivity.java:71]: Change the messgae to "Welcome" or something ?

**Review by ghost — CHANGES_REQUESTED** [review]
Please check the comments

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (agnesh02) [reply app/src/main/AndroidManifest.xml:19]: Shall change later

File: app/src/main/java/com/example/firebasetutorial/LoginActivity.java (hunk 2 above)
Comment (agnesh02) [reply app/src/main/java/com/example/firebasetutorial/LoginActivity.java:71]: Okay

**Review by ghost — APPROVED** [review]
Go ahead.

**Pull request merged**